### PR TITLE
Added handle_error for mdx_server, and logging configurations

### PR DESCRIPTION
**Pull request by anas-yousef**
## Related Issues
fixes: https://jira-dc.paloaltonetworks.com/browse/XSUP-32560

## Description
Added the **handle_error** option when checking if the mdx server is up fails.

### CODE REVIEW

File: demisto_sdk/commands/common/hook_validations/readme.py [hunk 1]
```diff
@@ -406,7 +413,9 @@ def is_docker_available():
             bool: True if the daemon is accessible
         """
         try:
-            docker_client: docker.DockerClient = init_global_docker_client(log_prompt="DockerPing")  # type: ignore
+            docker_client: docker.DockerClient = init_global_docker_client(
+                log_prompt="DockerPing"
+            )  # type: ignore
```
Comment (dorschw) [inline demisto_sdk/commands/common/hook_validations/readme.py:418]: 1. ignore a specific error
2. why ignore?

Comment (anas-yousef) [reply demisto_sdk/commands/common/hook_validations/readme.py:418]: Pre-commit did these changes

File: .changelog/4048.yml [hunk 2]
```diff
@@ -0,0 +1,4 @@
+changes:
+- description: Fixed an issue where **validate** would fail from trying to connect to MDX server without stating the reason.
```
Comment (dorschw) [inline .changelog/4048.yml:2]:
```suggestion
- description: Fixed an issue where **validate** would not mention the reasin when failing to connect to MDX.
```

File: demisto_sdk/commands/common/hook_validations/readme.py [hunk 3]
```diff
@@ -219,6 +219,15 @@ def is_valid_file(self) -> bool:
     def mdx_verify_server(self) -> bool:
         server_started = mdx_server_is_up()
         if not server_started:
+            if self.handle_error(
+                (
+                    "Validation of MDX file failed due to incapability of starting node server. You can skip this by adding RM103"
```
Comment (JudahSchwartz) [inline demisto_sdk/commands/common/hook_validations/readme.py:224]:
```suggestion
                    "Validation of MDX file failed due to unable to start the mdx server. You can skip this by adding RM103"
```

**Review by JudahSchwartz — APPROVED** [review]
you checked manually that this skip works as you described right?

**Pull request merged**